src/map/atcommand.c

```diff
@@ -1714,6 +1714,15 @@ ACMD(bodystyle)
 
 	memset(atcmd_output, '\0', sizeof(atcmd_output));
 
+	if (!((sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS || (sd->job&MAPID_THIRDMASK) == MAPID_GENETIC
```

Comment:
I stated that boolean isn't a type, but I'd like the conditions to be formally correct, regardless of the C type system.

I insist that the check is incorrect, regardless of its compliance with the C standard. Comparing the result of a logical or against zero makes no sense, even if it returns the correct result.

I'll be fixing it myself before merging.